examples/java/src/main/java/com/twilio/rest/api/v2010/credential/NewCredentials.java

```diff
@@ -0,0 +1,366 @@
+
+        @JsonCreator
+        public static Status forValue(final String value) {
+            return Promoter.enumFromString(value, Status.values());
+        }
+    }
+    public enum Permissions {
+        GET_ALL("get-all"),
+        POST_ALL("post-all");
+
+        private final String value;
+
+        private Permissions(final String value) {
+            this.value = value;
+        }
+
+        public String toString() {
+            return value;
+        }
+
+        @JsonCreator
+        public static Permissions forValue(final String value) {
+            return Promoter.enumFromString(value, Permissions.values());
+        }
+    }
+
+    private final String accountSid;
+    private final String sid;
+    private final String testString;
+    private final Integer testInteger;
+    private final PhoneNumberCapabilities testObject;
+    private final ZonedDateTime testDateTime;
+    private final BigDecimal testNumber;
+    private final Currency priceUnit;
+    private final Float testNumberFloat;
+    private final NewCredentials.TestEnum testEnum;
+    private final List<Integer> testArrayOfIntegers;
+    private final List<List<Integer>> testArrayOfArrayOfIntegers;
+    private final List<FeedbackIssue> testArrayOfObjects;
+    private final NewCredentials.XTwilioWebhookEnabled xTwilioWebhookEnabled;
+    private final NewCredentials.Status status;
+    private final NewCredentials.Permissions permissions;
+
+    @JsonCreator
+    private NewCredentials(
+        @JsonProperty("account_sid")
+        final String accountSid,
+
+        @JsonProperty("sid")
+        final String sid,
+
+        @JsonProperty("test_string")
+        final String testString,
+
+        @JsonProperty("test_integer")
+        final Integer testInteger,
+
+        @JsonProperty("test_object")
+        final PhoneNumberCapabilities testObject,
+
+        @JsonProperty("test_date_time")
+        final String testDateTime,
+
+        @JsonProperty("test_number")
+        final BigDecimal testNumber,
+
+        @JsonProperty("price_unit")
+        @JsonDeserialize(using = com.twilio.converter.CurrencyDeserializer.class)
+        final Currency priceUnit,
+
+        @JsonProperty("test_number_float")
+        final Float testNumberFloat,
+
+        @JsonProperty("test_enum")
+        final NewCredentials.TestEnum testEnum,
+
+        @JsonProperty("test_array_of_integers")
+        final List<Integer> testArrayOfIntegers,
+
+        @JsonProperty("test_array_of_array_of_integers")
+        final List<List<Integer>> testArrayOfArrayOfIntegers,
+
+        @JsonProperty("test_array_of_objects")
+        final List<FeedbackIssue> testArrayOfObjects,
+
+        @JsonProperty("x_twilio_webhook_enabled")
+        final NewCredentials.XTwilioWebhookEnabled xTwilioWebhookEnabled,
+
+        @JsonProperty("status")
+        final NewCredentials.Status status,
+
+        @JsonProperty("permissions")
+        final NewCredentials.Permissions permissions
+    ) {
```

Comment:
Nit: Unsure if this is the expected formatting?

Reply:
Formatting isn't a big deal. Didn't mess with the templates.